app/javascript/mastodon/features/annual_report/percentile.tsx

```diff
@@ -0,0 +1,43 @@
+import { FormattedMessage, FormattedNumber } from 'react-intl';
+
+import type { Percentiles } from 'mastodon/models/annual_report';
+
+export const Percentile: React.FC<{
+  data: Percentiles;
+}> = ({ data }) => {
+  const percentile = data.statuses;
+
+  return (
+    <div className='annual-report__bento__box annual-report__summary__percentile'>
+      <div className='annual-report__summary__percentile__label'>
+        <FormattedMessage
+          id='annual_report.summary.percentile.that_puts_you_in_the_top'
+          defaultMessage='That puts you in the top'
+        />
+      </div>
+      <div className='annual-report__summary__percentile__number'>
+        <FormattedNumber
+          value={percentile / 100}
+          style='percent'
+          maximumFractionDigits={1}
+        />
+      </div>
+      <div>
+        <div className='annual-report__summary__percentile__label'>
+          <FormattedMessage
+            id='annual_report.summary.percentile.of_mastodon_users'
+            defaultMessage='of Mastodon users.'
+          />
+        </div>
+        {percentile < 6 && (
+          <div className='annual-report__summary__percentile__footnote'>
+            <FormattedMessage
+              id='annual_report.summary.percentile.we_wont_tell_bernie'
+              defaultMessage="We won't tell Bernie."
+            />
+          </div>
+        )}
+      </div>
```

Comment:
What if the whole thing was one translatable string with various placeholders?

Something like (untested):

```jsx
<FormattedMessage
  id='annual_report.summary.percentile_text'
  defaultMessage='<label>That puts you in the top</label><percentage /><label>of Mastodon users.</label>'
  values={{
   label: (text) => <div className='annual-report__summary__percentile__label'>{text}</div>,
   percentage: () => <div className='annual-report__summary__percentile__number'><FormattedNumber value={percentile / 100} style='percent' maximumFractionDigits={1} /></div>,
  }}
/>
```

Reply:
Okay, it's a bit clunky but it works.